workflow/scripts/plot_annotatepeaks_summary_homer.R

```diff
@@ -0,0 +1,18 @@
+log <- file(snakemake@log[[1]], open="wt")
+sink(log)
+sink(log, type="message")
+
+library("tidyverse")
+
+homer_data <- read_tsv(snakemake@input[[1]])
+homer_data <- homer_data %>% gather(`exon`, `Intergenic`, `intron`, `promoter-TSS`, `TTS`, key="sequence_element", value="counts")
```

Comment:
Cool, that this now works. And the whole plotting script is rather straightforward, this way!

Reply:
yes, many thanks for your help :)